### DISCUSSION TITLE
How to draw a sensitivity analysis spider plots？

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1673] [Questions & Problems]
![微信截图_20231122140034](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/109507330/ddee5cb6-400c-4870-972d-65caf8f49dbc)

How to draw a sensitivity analysis spider plot like the one shown above? How can variables on the X-axis be modified from PK sim?

**Comment by PavelBal** [comment]
PK-Sim/MoBi do not offer such type of plots (yet). You can use the `esqlabsR` R package do perform sensitivity analysis and create such a figure as described here: https://esqlabs.github.io/esqlabsR/articles/esqlabsR-sensitivity.html
